helm/capa-iam-operator/values.yaml

```diff
@@ -26,10 +26,14 @@ verticalPodAutoscaler:
 
 # Add seccomp to pod security context
 podSecurityContext:
+  runAsNonRoot: true
   seccompProfile:
     type: RuntimeDefault
 
 # Add seccomp to container security context
 securityContext:
   seccompProfile:
     type: RuntimeDefault
+  capabilities:
```

Comment:
Could we add defaults to schema? I already left this comment in another PR, sorry for the repetition.